### ISSUE TITLE
Python interface

### ISSUE THREAD

**Issue by tovbinm** [salesforce/TransmogrifAI#393]
**Problem**
TransmogrifAI is currently only usable from Scala with Spark. It would be great if one could:
1. Load TransmogrifAI models in Python, display model insights and compare/evaluate with other ml libraries
2. Define TransmogrifAI workflows from Python, train and save the model
3. Define new stages through UDFs from Python 

**Solution**
Let's discuss the plan on how we would like to prioritize and implement it.

**Additional Context**
We would need to think about how we standardize model format (perhaps double down on MLeap?) for (1). For (2) we would need to pick which stages we want to expose first and

**Comment by tovbinm** [comment]
Yeah, we were planning to add it - #394  . Mot likely nobody is working on it yet. @wsuchy @leahmcguire please confirm.

**Comment by UnixJunkie** [comment]
It would be amazing if the whole thing, including model training, would be usable from Python.

PS: if you make it usable from OCaml, I would also be happy, but that probably makes less new happy users than Python wrappers

**Comment by tovbinm** [comment]
@leahmcguire @gerashegalov correct me if I am wrong, but I don't think anyone is working on neither Pything nor OCaml interface this an this point. That's mainly due to the fact that TransmogrifAI is being used from Scala codebase & notebooks internally.
